workspace-hack/Cargo.toml

```diff
@@ -26,7 +26,8 @@ getrandom = { version = "0.2", default-features = false, features = ["js", "rdra
 hashbrown = { version = "0.14", default-features = false, features = ["raw"] }
 hyper = { version = "0.14", features = ["full"] }
 indexmap = { version = "1", default-features = false, features = ["std"] }
-itertools = { version = "0.10" }
+itertools-93f6ce9d446188ac = { package = "itertools", version = "0.10" }
+itertools-a6292c17cd707f01 = { package = "itertools", version = "0.11" }
```

Comment:
Can we unify these two itertools versions?

Reply:
Probably not, this is generated by Hakari.

Reply:
0.10 is what we are using, 0.11 is used by some dependent libraries. Do we need to update the itertools version?
Its latest version is 0.12.

Reply:
updated to 0.11